pysgrid/utils.py

```diff
@@ -171,7 +171,7 @@ def infer_variable_location(sgrid, variable):
         edge_dims = []
     var_dims = variable.dimensions
     if (does_intersection_exist(var_dims, face_dims) and not
-       does_intersection_exist(var_dims, node_dims)):
+            does_intersection_exist(var_dims, node_dims)):
```

Comment:
Please avoid moving code around. That produces a lot of noise for the reviewers.  If that is necessary for some style guide do it in a separate PR.